icu4j/main/tests/core/src/com/ibm/icu/dev/test/number/DecimalQuantityTest.java

```diff
@@ -603,8 +608,268 @@ public void testNickelRounding() {
+                fn.getFixedDecimal();
+
+        double expectedNOperand = 1234.5;
+        double expectedIOperand = 1234;
+        double expectedFOperand = 50;
+        double expectedTOperand = 5;
+        double expectedVOperand = 2;
+        double expectedWOperand = 1;
+        double expectedEOperand = 3;
+        String expectedString = "1,23450 millier";
+        double actualNOperand = dq.getPluralOperand(Operand.n);
+        double actualIOperand = dq.getPluralOperand(Operand.i);
+        double actualFOperand = dq.getPluralOperand(Operand.f);
+        double actualTOperand = dq.getPluralOperand(Operand.t);
+        double actualVOperand = dq.getPluralOperand(Operand.v);
+        double actualWOperand = dq.getPluralOperand(Operand.w);
+        double actualEOperand = dq.getPluralOperand(Operand.e);
+        String actualString = fn.toString();
+
+        assertDoubleEquals(
+                String.format("compact decimal fraction n operand: %f", inputVal),
+                expectedNOperand,
+                actualNOperand);
+        assertDoubleEquals(
+                String.format("compact decimal fraction i operand: %f", inputVal),
+                expectedIOperand,
+                actualIOperand);
+        assertDoubleEquals(
+                String.format("compact decimal fraction f operand: %f", inputVal),
+                expectedFOperand,
+                actualFOperand);
+        assertDoubleEquals(
+                String.format("compact decimal fraction t operand: %f", inputVal),
+                expectedTOperand,
+                actualTOperand);
+        assertDoubleEquals(
+                String.format("compact decimal fraction v operand: %f", inputVal),
+                expectedVOperand,
+                actualVOperand);
+        assertDoubleEquals(
+                String.format("compact decimal fraction w operand: %f", inputVal),
+                expectedWOperand,
+                actualWOperand);
+        assertDoubleEquals(
+                String.format("compact decimal fraction e operand: %f", inputVal),
+                expectedEOperand,
+                actualEOperand);
+        assertEquals(
+                String.format("compact decimal fraction toString: %f", inputVal),
+                expectedString,
+                actualString);
+    }
+
+    @Test
+    public void testSuppressedExponentUnchangedByInitialScaling() {
+        ULocale locale = new ULocale("fr-FR");
+        LocalizedNumberFormatter withLocale = NumberFormatter.withLocale(locale);
+        LocalizedNumberFormatter compactLong =
+                withLocale.notation(Notation.compactLong());
+        LocalizedNumberFormatter compactScaled =
+                compactLong.scale(Scale.powerOfTen(3));
+
+        Object[][] casesData = {
+                // formatter name, input, string output,
+                // compact n operand, compact i operand, compact e operand,
+                // is compact e operand same when in simple notation
```

Comment:
I think these descriptions are out of date?

Reply:
Done.